### ISSUE TITLE
Access to native pickers

### ISSUE THREAD

**Issue by ferllings** [mockingbot/electron-color-picker#5]
Hello,

Is it possible to get access to the OSX/Windows native pickers source code?
I would like to add additional features, like keyboard navigation, and see if this is possible to also return the picker position, (to detect the display, and find the color profile for color conversion)

Thanks

**Comment by ThatBean** [comment]
The repo is at [mockingbot/mb_colorpicker_desktop_native](https://github.com/mockingbot/mb_colorpicker_desktop_native),
and the code kept minimal to just do the job.

If more additional features is needed and bloat is OK, you may check the previous Qt version at [mockingbot/mb_colorpicker_desktop_qt](https://github.com/mockingbot/mb_colorpicker_desktop_qt).

**Comment by ferllings** [comment]
Thanks, I agree this should stay minimal.
But the picker works in sRGB color space, which returns wrong results if your display is not in sRGB. So I believe an option/way to adjust that should be included.
I'll try to provide a PR in the next few weeks.

**Comment by ThatBean** [comment]
That's a problem we did not fully solve, it'll be great if you can work it out.

In our use case, mainly it's for getting color from browser. But last time we checked, the same CSS color in different browser displays different on screen. Maybe it's software related too, and there's so questions like [this](https://stackoverflow.com/questions/30050879/color-picker-on-mac-os-x-selects-wrong-colors).

**Comment by ferllings** [comment]
Yes, I'm well aware of this issue. Even with browsers you can get different color results: Firefox use sRGB color space, while Chrome and Safari use the display colorSpace.
And if, like me, you have 2 displays, with 2 diff. colorSpace2, and you test on multiple browsers, that's a nightmare to get accurate results :)
So it's not possible to fix this because the picker can't know what browser he's hovering, but at least I can try to make it possible for the user to choose the colorSpace for the picker to operate, or, to convert it to.
